week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,16 +1,18 @@
 // Add comments to explain what this function does. You're meant to use Google!
-function getNumber() {
-    return Math.random() * 10;
+
+function getNumber() { 
+    return Math.random() * 10;// this funtion returns a random number then multiplys by 10.
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
 function s(w1, w2) {
-    return w1.concat(w2);
+    return w1.concat(w2);// w1 and w2 both are seprate arrays .concat merges array w2 with w1. 
```

Comment:
Good answer. For extra points to show deeper understanding, explain whether w1.concat(w2):
- returns a brand new array made up of the merging of w1 and w2
- w1 is updated with the new merged array
- or w2 is updated with the new merged array